### PR TITLE
📦 NEW: Reverse order of output

### PR DESCRIPTION
**Pull request by asharirfan**
## Description

This PR adds `--reverse` flag to the CLI, which reverses the order of output.

## Usage

```sh
corona --reverse
corona --sort cases --reverse
corona -s cases -r
```

## Applicable Issues

#23

### CODE REVIEW

**Comment by ahmadawais** [comment]
Thank you, Ashar!

Please update the PR in light of the latest release.

🔗 https://github.com/ahmadawais/corona-cli/releases/tag/3.4.0
🔗 https://www.npmjs.com/package/corona-cli

**Comment by asharirfan** [comment]
Just updated the PR with the latest release.

File: utils/cli.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ module.exports = meow(
 	Options
 	  ${yellow(`--xcolor`)}, ${yellow(`-x`)}         Single colored output
 	  ${yellow(`--sort`)}, ${yellow(`-s`)}           Sort data by type
+	  ${yellow(`--reverse`)}, ${yellow(`-r`)}        Reverse the order of sorted data
```
Comment (ahmadawais) [inline utils/cli.js:19]: Smaller description is better.

File: utils/cli.js [hunk 2]
```diff
@@ -41,6 +42,10 @@ module.exports = meow(
 				type: "string",
 				default: "cases",
 				alias: "s"
+			},
+			reverse: {
+				type: "string",
```
Comment (ahmadawais) [inline utils/cli.js:47]: You didn't define a default.

File: utils/getCountries.js [hunk 3]
```diff
@@ -15,7 +15,11 @@ module.exports = async (spinner, table, states, countryName, sortBy) => {
 		let allCountries = response.data;
 
 		// Sort.
-		allCountries = orderBy(allCountries, [sortingKeys[sortBy]], ["desc"]);
+		if ( reverse !== undefined ) {
```
Comment (ahmadawais) [inline utils/getCountries.js:18]: The ternary operator would have been better here.

File: utils/getCountries.js [hunk 4]
```diff
@@ -35,6 +39,9 @@ module.exports = async (spinner, table, states, countryName, sortBy) => {
 
 		spinner.stopAndPersist();
 		spinner.info(`${chalk.cyan(`Sorted by:`)} ${sortBy}`);
+		if ( reverse !== undefined ) {
+			spinner.info(`${chalk.cyan(`Order:`)} reverse`);
```
Comment (ahmadawais) [inline utils/getCountries.js:43]: Too much info.

**Comment by ahmadawais** [comment]
Thank you, Ashar. Accepting the PR but do read the comments. I'll address them myself.

**Comment by ahmadawais** [comment]
Also why is reverse a string?

**Comment by ahmadawais** [comment]
Done

🔗 https://github.com/ahmadawais/corona-cli/releases/tag/3.5.0
🔗 https://www.npmjs.com/package/corona-cli

Check out the final code.

**Comment by alf946** [comment]
On:
`$ corona -r -s cases -l 10 `
produces (greatest number) top 10 with reverse sorting, not the (least number) top 10 sorted least to most.

**Pull request merged**